packages/gitbook-docs/src/scripts/replaceEscapedTokensWithBackTicks.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable functional/immutable-data */
+/* eslint-disable functional/no-let */
+/* eslint-disable functional/no-loop-statements */
+/* eslint-disable functional/no-expression-statements */
+/* eslint-disable functional/no-try-statements */
+import { DOCUMENTATION_PATH } from '../helpers/documentationParsing.helper';
+import path from 'path';
+import fs from 'fs';
+
+export async function replaceEscapedTokensWithBackTicks(
+  filePath: string
+): Promise<string> {
+  let depth = 0;
+  const fileContent = [...(await fs.promises.readFile(filePath, 'utf8'))];
+  for (const index in fileContent) {
+    if (fileContent[index] === '\\' && fileContent[Number(index) + 1] === '<') {
```

Comment:
Why is the second one cast to number and the first is not?
```suggestion
    if (fileContent[Number(index)] === '\\' && fileContent[Number(index) + 1] === '<') {
```